vnext/Microsoft.ReactNative/ABIViewManager.cpp

```diff
@@ -215,6 +216,56 @@ YGMeasureFunc ABIViewManager::GetYogaCustomMeasureFunc() const {
   }
 }
 
+void ABIViewManager::OnPointerPressed(
```

Comment:
We could consolidate these into an `OnPointerEvent` method if we added an enum for PointerEventType to the ReactPointerEventArgs

Reply:
It probably makes sense to unify to a single event since at least two of our use cases (Text hit testing and pointerEvents="box-none" | "box-only") have the same logic regardless of the pointer event type.

Reply:
I imagine many pointer event handling cases will be "uniform" across event types, and having to add 5 methods is annoying to say the least.